src/pages/Options/Options.tsx

```diff
@@ -150,9 +139,9 @@ const Options = (): JSX.Element => {
             </p>
             <p>
               GitHub:{' '}
-              <Link href={HYPERCRX_GITHUB} target="_blank" underline>
+              <a href={HYPERCRX_GITHUB} target="_blank">
```

Comment:
Do you think it would be better if we use antd's Button with 'link' to replace <a/>, it would be easier to control its style and behavior. Something like this:
``` ts
<Button type="text">Text</Button>
```

Reply:
I feel like it might be more appropriate to keep the link as is?